lib/fog/profitbricks/models/compute/server.rb

```diff
@@ -175,7 +176,7 @@ def list_volumes
           volumes = result.body['items'].each { |volume| volume['datacenter_id'] = datacenter_id }
           result.body['items'] = volumes
 
-          result.body
+          result.body['items']
```

Comment:
Thank you for your feedback.

This was reverted in the latest commit. It was an oversight on my end.